### PR TITLE
December network release -- ARM feedback

### PR DESCRIPTION
**Pull request by seyedfayaz**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/network/resource-manager/Microsoft.Network/stable/2018-11-01/ddosCustomPolicy.json [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
```
Comment (ravbhatnagar) [inline specification/network/resource-manager/Microsoft.Network/stable/2018-11-01/ddosCustomPolicy.json]: PATCH missing. Looks like this is a tracked resource and so PATCH support is required.

Comment (ravbhatnagar) [inline specification/network/resource-manager/Microsoft.Network/stable/2018-11-01/ddosCustomPolicy.json]: List API to list all ddosCustomPOlicies under the resource group and under the subscription are missing.

**Review by jianghaolu — CHANGES_REQUESTED** [review]

File: specification/network/resource-manager/Microsoft.Network/stable/2018-11-01/ddosCustomPolicy.json [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+          "202": {
+            "description": "Accepted and the operation will complete asynchronously."
+          },
+          "200": {
+            "description": "Delete successful."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
```
[the remark this answers is not in the document]
Comment (seyedfayaz) [reply specification/network/resource-manager/Microsoft.Network/stable/2018-11-01/ddosCustomPolicy.json]: def added

File: specification/network/resource-manager/Microsoft.Network/stable/2018-11-01/ddosCustomPolicy.json (hunk 1 above)
Comment (seyedfayaz) [reply specification/network/resource-manager/Microsoft.Network/stable/2018-11-01/ddosCustomPolicy.json]: could you clarify please?

Comment (KrisBash) [reply specification/network/resource-manager/Microsoft.Network/stable/2018-11-01/ddosCustomPolicy.json]: @seyedfayaz all tracked resources (that support tags) need a PATCH api that supports granular property update, at least covering the tags property. See the [RPC](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#patch-resource)

**Review by jianghaolu — CHANGES_REQUESTED** [review]

File: specification/network/resource-manager/Microsoft.Network/stable/2018-11-01/examples/DdosCustomPolicyCreate.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
+{
+  "parameters": {
+    "api-version": "2018-11-01",
+    "subscriptionId": "subid",
+    "resourceGroupName": "rg1",
+    "ddosCustomPolicyName": "test-ddos-custom-policy",
+    "parameters": {
+      "properties": {
+          "protocol": "Tcp"
```
[the remark this answers is not in the document]
Comment (mancgupta) [reply specification/network/resource-manager/Microsoft.Network/stable/2018-11-01/examples/DdosCustomPolicyCreate.json]: Updated.

File: specification/network/resource-manager/Microsoft.Network/stable/2018-11-01/ddosCustomPolicy.json (hunk 1 above)
Comment (mancgupta) [reply specification/network/resource-manager/Microsoft.Network/stable/2018-11-01/ddosCustomPolicy.json]: We are not currently supporting the list API. We are planning to add this as a part of next version.

Comment (mancgupta) [reply specification/network/resource-manager/Microsoft.Network/stable/2018-11-01/ddosCustomPolicy.json]: I have added the PATCH API to update the tags

**Review by jianghaolu — APPROVED** [review]

**Pull request merged**